lib/msf/core/auxiliary/login.rb

```diff
@@ -35,7 +35,7 @@ def create_login_ivars
     #
     # Some of these regexes borrowed from NeXpose, others added from datasets
     #
-    @login_regex = /(?:log[io]n( name|)|user(name|id|))\s*\:/i
+    @login_regex = /(?:log[io]n( name|)|user( ?name|id|))\s*\:/i
```

Comment:
Would anyone have a problem with generalizing the suffix/space handling like so?

/(?:(log[io]n|user) ?(name|id|))\s*\:/i